mhkit/wave/io/wave_hindcast.py

```diff
@@ -0,0 +1,84 @@
+import pandas as pd
+import numpy as np
+from rex import WaveX, MultiYearWaveX
+
+def read_US_wave_dataset(wave_path, parameter, lat_lon, years=None, tree=None, 
```

Comment:
I think we should consider a different name for this function. We tend to use read for local data and request for hosted data. I think we know this is wave as it is in the `mhkit.wave` module so I would suggest removing this redundant word. Does this dataset have a more formal name?

Reply:
addressed in https://github.com/MHKiT-Software/MHKiT-Python/pull/60/commits/798812b0ac650281dc7fcaf19db820f623b90bf7